Source/Public/Remove-VSTeamAccessControlEntry.ps1

```diff
@@ -0,0 +1,76 @@
+function Remove-VSTeamAccessControlEntry {
+    [CmdletBinding(DefaultParameterSetName = 'ByNamespace')]
+    param(
+       [Parameter(ParameterSetName = 'ByNamespace', Mandatory = $true, ValueFromPipeline = $true)]
+       [VSTeamSecurityNamespace] $SecurityNamespace,
+ 
+       [Parameter(ParameterSetName = 'ByNamespaceId', Mandatory = $true)]
+       [ValidateScript({
+          try {
+              [System.Guid]::Parse($_) | Out-Null
+              $true
+          } catch {
+              $false
+          }
+       })]
+       [string] $SecurityNamespaceId,
```

Comment:
If you change the type to [guid] you should not need the verification script. I just searched the code base and found other examples where I let this slip through. Change this to [guid] and I will issue PRs to correct those other areas as well.

Reply:
Changed to [GUID] from [String] in latest commit.